www/styles/Nav.module.css

```diff
@@ -26,6 +27,7 @@
   font-size: 2em;
   list-style: none;
   font-weight: bold;
+  color: #9dbeff
```

Comment:
Do we need this color, maybe we should add it as a global. That way if we wan't to add themes we can change it in one place.

Reply:
I didn't actually know the proper way to make everything in the nav bar the color we want it.